packages/scxa-faceted-search-results/html/fetch-loader/ExperimentCard.js

```diff
@@ -58,7 +66,7 @@ class ExperimentCard extends React.Component {
   }
 
   render() {
-    const {url, species, experimentDescription, markerGenes, numberOfAssays, factors} = this.props
+    const {url, species, experimentDescription, markerGenes, numberOfAssays, experimentAccession, type, factors} = this.props
```

Comment:
I might miss something, but I can't see that the `experimentAccession` and `type` is being added to the `props`. Am I wrong?